Asterix/utils/hci_metrics.py

```diff
@@ -250,3 +250,44 @@ def contrast_curves(reduced_data, xcen=None, ycen=None, delta_raddii=3, type_of_
     contrast_curve = np.asarray(contrast_curve)
     contrast_curve[np.where(contrast_curve == 0)] = np.nan
     return contrast_curve
+
```

Comment:
is this useful ? I might be wrong but it seems that you only want to use the existing `plot_contrast_curves`